en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -36,23 +40,113 @@ often working alongside Core Committers.
 
 CMS Squad members have write access to core repositories in order to work effectively with GitHub issues. They are expected to use those permissions with good judgement for merging pull requests.
 
+### Peer Reviewers
+
+[info]
+This is a new role, and is currently in a trial period. It is likely to change over time as we learn what works and what doesn't.
```

Comment:
I'm happy for any cards to be created that you feel like need to be created. It feels like you have a specific idea in mind for the card so it would probably be better for you to create it. A card I create based on your suggestion would likely miss a lot of context that you have in your head.

Reply:
https://github.com/silverstripe/developer-docs/issues/414